### DISCUSSION TITLE
How to handle changes to submodule files?

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#1794] [Developers] [answered]
The RtAudio file is changed as of #1788. Now when I check the status of git for my fork it's reported as changed. How am I supposed to handle it? Should I add it?

**Comment by oleg68** [comment, marked as answer]
Does ``git submodule update --init --recursive`` help?

**Reply by larspalo** [reply]
@oleg68 Yes, that does indeed remove the red notice of "git status". But doesn't that update/change to the latest commit of the submodule project? Not that it matters so much for me on Linux as the GO change to RtAudio relates to windows.

**Reply by oleg68** [reply]
> But doesn't that update/change to the latest commit of the submodule project?

No, it does not. It updates the submodel directories to the commits specified at the main (Grandorgue) repository, not ti the latest ones.

**Comment by oleg68** [comment]
> But doesn't that update/change to the latest commit of the submodule project?

No, it does not. It updates the submodel directories to the commits specified at the main (Grandorgue) repository, not ti the latest ones.

**Reply by larspalo** [reply]
Ok. Thanks for explaining!
